properties-parser.ts

```diff
@@ -10,6 +10,19 @@ export class PropertiesParser implements IPropertiesParser {
 	public parse(text: string): any {
 		return propertiesParser.parse(text);
 	}
+	
+	public read(filePath: string): IFuture<IStringDictionary> {
+		let future = new Future<any>();
```

Comment:
The return type declaration of the function and the return variable are different - which is the correct one?